src/tests/screens/SurveyDetail/detail.test.js

```diff
@@ -22,7 +22,9 @@ describe('When Detail component is mounted', () => {
         </DetailsContext.Provider>
       </Router>
     );
-    const surveyImage = container.querySelector('img[class="survey-detail__image"]');
+    const surveyImage = container.querySelector(
```

Comment:
@rafayet-monon As I add a comment here (https://github.com/rafayet-monon/surveyor/pull/20/files#r544779215), I think we need more test for all types of question we have ✌️

Reply:
@Lahphim I plan to add more test codes when I add functionalities from the API response. These PRs are just for component implementation and design.